datasets/csv/csv.py

```diff
@@ -2,7 +2,6 @@
 
 from dataclasses import dataclass
 
-import pyarrow as pa
```

Comment:
Deleted this import since it doesn't seem to be in use @thomwolf

Reply:
Ok!